src/ext/java/org/opentripplanner/ext/vectortiles/LayerBuilder.java

```diff
@@ -0,0 +1,60 @@
+package org.opentripplanner.ext.vectortiles;
+
+import com.wdtinc.mapbox_vector_tile.VectorTile;
+import com.wdtinc.mapbox_vector_tile.adapt.jts.JtsAdapter;
+import com.wdtinc.mapbox_vector_tile.adapt.jts.TileGeomResult;
+import com.wdtinc.mapbox_vector_tile.build.MvtLayerBuild;
+import com.wdtinc.mapbox_vector_tile.build.MvtLayerParams;
+import com.wdtinc.mapbox_vector_tile.build.MvtLayerProps;
+import org.locationtech.jts.geom.Envelope;
+import org.locationtech.jts.geom.Geometry;
+import org.locationtech.jts.geom.GeometryFactory;
+import org.opentripplanner.common.geometry.GeometryUtils;
+
+import java.util.List;
+
+public abstract class LayerBuilder<T> {
```

Comment:
Add some JavaDoc here mention that this can be reused; henced moved to the core when there is 2 sub-classes of this.